README.md

```diff
@@ -5,9 +5,23 @@
 
 **Minimalistic init system for containers injecting secrets from various secret stores.**
 
-## Usage
+## Features
 
-TODO
+
+- `Multi-provider support`: No need to explicitly specify the secret provider; instead, secrets are loaded based on environment variable references.
+- `Async loading`: Secrets are loaded asynchronously, enhancing speed and efficiency.
+- `Load without modification`: `Secret-init` replaces environment variable values, with values from secret providers, without modification.
+- `Renew secrets`: If `secret-init` is used in daemon mode, it will renew secrets in the background.
+
+| **Supported Providers**                                      | **Status**     |
+|--------------------------------------------------------------|----------------|
+| [HashiCorp Vault](https://www.vaultproject.io)               | ✅ Implemented |
+| [AWS Secrets Manager](https://aws.amazon.com/secrets-manager)| Upcoming       |
+
+## Getting started
+
+- `Secret-init` is designed for use with the [Kubernetes mutating webhook](https://bank-vaults.dev/docs/mutating-webhook/); however, it can also function as a standalone tool.
```

Comment:
```suggestion
- `secret-init` is designed for use with the [Kubernetes mutating webhook](https://bank-vaults.dev/docs/mutating-webhook/). It can also function as a standalone tool.
```